stdlib/http/src/main/java/org/ballerinalang/net/http/BallerinaHTTPConnectorListener.java

```diff
@@ -72,6 +72,9 @@ public void onMessage(HttpCarbonMessage inboundMessage) {
             httpResource = HttpDispatcher.findResource(httpServicesRegistry, inboundMessage);
             if (HttpDispatcher.shouldDiffer(httpResource)) {
                 inboundMessage.setProperty(HTTP_RESOURCE, httpResource);
+                //Removes inbound content listener since data binding waits for all contents to be received
+                //before executing its logic.  
+                inboundMessage.removeInboundContentListener();
```

Comment:
Shall we add a test case for this change?

Reply:
This needs at least a 2.5MB payload and currently we cannot configure this value. If we are to write a test we need to keep atleast a 2.5MB file in ballerina. Therefore I suggest we add the test case once we have a way to configure this inbound trottling value.